HLTrigger/special/plugins/HLTMuonRechitClusterFilter.cc

```diff
@@ -0,0 +1,130 @@
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "HLTrigger/HLTcore/interface/HLTFilter.h"
+
+#include "DataFormats/MuonReco/interface/MuonRecHitCluster.h"
```

Comment:
Please make sure to update ` HLTrigger/special/plugins/BuildFile.xml` accordingly.